.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
   "arrowParens": "avoid",
-  "semi": false
-}
+  "semi": false,
```

Comment:
somehow when I ran `yarn format` I got so many changes and this reduced it a little.